server/routes/api/results.js

```diff
@@ -0,0 +1,28 @@
+//results.js
+
+const express = require("express");
+const router = express.Router();
+
+//External Modules
+import { getVotes } from "../utils/getVotes";
+
+router.get("/results", (req, res) => {
+    if (req.body === null) {
+        res.send({ status: 400, error: "No poll id provided" });
```

Comment:
Can we please agree to send errors as `res.status(<status number>).json({ error: 'some error' })`. This way all the responses with status code not equal to 200 will be caught in the catch() block and we don't need to check the status in then() block as `response.data.status === 500`. Thank you! I will need to rewrite my code too.